src/main/java/Entities/UserMatches/GroupScore.java

```diff
@@ -0,0 +1,79 @@
+package Entities.UserMatches;
+
+import Entities.Group;
+import Entities.User;
+import Entities.UserPublicProfile;
+
+import java.util.HashMap;
+
+/**
+ * General Description:
+ * This class is used to store the User, Group, and their similarity score
+ *
+ * ATTRIBUTES:
+ * user: the Current User
+ * group: the Group being compared
+ * score: the similarity score between User and Group calculated using the method calculateSimilarity()
+ *
+ * METHOD DESCRIPTIONS:
+ * compareTo(): compares the score of this and another GroupScore
+ *
+ * calculateSimilarity(): calculates and returns the similarity score based on the UserPublicProfile preferences
+ * of user and the preferences of group
+ */
+
```

Comment:
Nice description! This makes it really easy to understand and follow your code. Thanks for making the such clear comments :)

Reply:
Agree with Sohee ^ setting the bar for doc strings 🥇  woooo